src/azul/project/hca/transformers.py

```diff
@@ -540,3 +558,20 @@ def get_aggregator(self, entity_type):
 
     def entity_type(self) -> str:
         return 'bundles'
+
+    def transform(self,
+                  uuid: str,
+                  version: str,
+                  manifest: List[JSON],
+                  metadata_files: Mapping[str, JSON]
+                  ) -> Sequence[Document]:
+        for contrib in super().transform(uuid, version, manifest, metadata_files):
+            # noinspection PyArgumentList
+            if 'project.json' in metadata_files:
+                # we can't handle v5 bundles
+                metadata = []
+            else:
+                metadata_generator = MetadataGenerator()
+                metadata_generator.add_bundle_files_to_row(uuid, version, [k for k in metadata_files.values()])
```

Comment:
```suggestion
                metadata_generator.add_bundle_files_to_row(uuid, version, list(metadata_files.values()))
```

Those things should be 2nd nature to you.

Also consider

```suggestion
                metadata_generator.add_bundle_files_to_row(uuid, version, metadata_files.values())
```

and changing the parameter to `Iterable[JSON]`